.travis.yml

```diff
@@ -25,30 +25,44 @@ jobs:
         - docker info
     - stage: tests
       script:
-        - ./vendor/bin/phpunit --testsuite base-tests --coverage-clover build/logs/clover.xml --debug 
+        - ./vendor/bin/phpunit --testsuite base-tests --coverage-clover build/logs/clover.xml --debug
         - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-        - chmod +x ./cc-test-reporter        
+        - chmod +x ./cc-test-reporter
         - ./cc-test-reporter format-coverage --output "coverage/codeclimate.1.json"
         - docker run --rm -e AWS_ACCESS_KEY_ID=$AWS_ECR_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_ECR_SECRET_ACCESS_KEY -v $TRAVIS_BUILD_DIR/coverage/:/data/ quay.io/keboola/aws-cli:latest s3 sync /data/ "s3://docker-runner-logs/coverage/$TRAVIS_BUILD_NUMBER"
     - stage: tests
       script:
         - ./vendor/bin/phpunit --testsuite runner-tests --coverage-clover build/logs/clover.xml --debug
         - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-        - chmod +x ./cc-test-reporter        
+        - chmod +x ./cc-test-reporter
         - ./cc-test-reporter format-coverage --output "coverage/codeclimate.2.json"
         - docker run --rm -e AWS_ACCESS_KEY_ID=$AWS_ECR_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_ECR_SECRET_ACCESS_KEY -v $TRAVIS_BUILD_DIR/coverage/:/data/ quay.io/keboola/aws-cli:latest s3 sync /data/ "s3://docker-runner-logs/coverage/$TRAVIS_BUILD_NUMBER"
     - stage: tests
       script:
         - ./vendor/bin/phpunit --testsuite executor-tests --coverage-clover build/logs/clover.xml --debug
         - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-        - chmod +x ./cc-test-reporter        
+        - chmod +x ./cc-test-reporter
+        - ./cc-test-reporter format-coverage --output "coverage/codeclimate.3.json"
+        - docker run --rm -e AWS_ACCESS_KEY_ID=$AWS_ECR_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_ECR_SECRET_ACCESS_KEY -v $TRAVIS_BUILD_DIR/coverage/:/data/ quay.io/keboola/aws-cli:latest s3 sync /data/ "s3://docker-runner-logs/coverage/$TRAVIS_BUILD_NUMBER"
+    - stage: tests
+      script:
+        - ./vendor/bin/phpunit --testsuite s3-tests --coverage-clover build/logs/clover.xml --debug
+        - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        - chmod +x ./cc-test-reporter
+        - ./cc-test-reporter format-coverage --output "coverage/codeclimate.3.json"
```

Comment:
```suggestion
        - ./cc-test-reporter format-coverage --output "coverage/codeclimate.4.json"
```

Reply:
a dole 4 a jeste podtim je parts a tam bude 5

Reply:
ale skoro bych rekl, ze je to stejne nejaky rozbity a nefunguje to (jinak by to melo rvat, ze klesnul coverage)

Reply:
aha, ok, upravim to, to som si nevsimol.

Reply:
no ja taky az na potreti